network/wsNetwork.go

```diff
@@ -752,6 +756,11 @@ func (wn *WebsocketNetwork) setup() {
 		wn.slowWritingPeerMonitorInterval = slowWritingPeerMonitorInterval
 	}
 
+	var seed crypto.Seed
+	crypto.RandBytes(seed[:])
+	wn.identityKeys = crypto.GenerateSignatureSecrets(seed)
+	wn.peersByID = map[crypto.PublicKey]*wsPeer{}
+
```

Comment:
I had it here in setup because that's where other random identity bits get set, but these are all fair questions. Presumably, if the network is ever restarted, it would be _important_ to have a new identity key, so that you don't get deduplicated inappropriately.

I'll shift it in, thanks!